ui/components/app/nfts-items/nfts-items.js

```diff
@@ -192,7 +192,7 @@ export default function NftsItems({
                 ipfsGateway,
               );
               const nftImageAlt = getNftImageAlt(nft);
-              const isImageHosted = image.startsWith('https:');
+              const isImageHosted = image?.startsWith('https:');
```

Comment:
I'm uncertain why this is now needed

Reply:
Seems this was just missed as optional chaining on `image` is also used here

https://github.com/MetaMask/metamask-extension/blob/0139b9913b762adf448d81f48d265a8fbc0abfd3/ui/components/app/nft-details/nft-details.js#L91

going to mark this as resolved

Reply:
Unresolving just to confirm that the missing optional chaining seems to have introduced a bug in 11.1.0 that this PR now resolves